### DISCUSSION TITLE
Outline access keys

### DISCUSSION THREAD

**Discussion by ParsaSr** [OutlineFoundation/outline-sdk#119] [General]
Hi everyone!
I am having a problem with access keys.
whenever I change my server, every single key that I had made gets destroyed.
I wish I could take backups of these access keys so I could add them in my new server.
does anyone know the Outline keys directory address inside the server?

**Comment by fortuna** [comment]
Moved to https://github.com/Jigsaw-Code/outline-server/issues/1434
